### PR TITLE
Give some love

### LINKED ISSUE #17: Releases should have a changelog
This would make it easier to know what is changed between versions

### PR DESCRIPTION
**Pull request by odolbeau**
Changelog, tests, badges, permissions.

More tests to come later. :)

Fix #17

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    <testsuites>
+        <testsuite name="RabbitMQ admin toolkit full test suite">
+            <directory>tests/</directory>
+        </testsuite>
+    </testsuites>
+
+    <filter>
+        <whitelist>
+            <directory>./</directory>
```
Comment (stof) [inline phpunit.xml.dist]: you should include only the `src` foldeR. Collecting code coverage on test files is useless

**Review by stof — CHANGES_REQUESTED** [review]

**Comment by stof** [comment]
@odolbeau looks like you messed something here

**Comment by odolbeau** [comment]
@stof it's better with the correct branch...

File: phpunit.xml.dist [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <testsuite name="RabbitMQ admin toolkit full test suite">
+            <directory>tests/</directory>
+        </testsuite>
+    </testsuites>
+
+    <filter>
+        <whitelist>
+            <directory>./src</directory>
+            <exclude>
+                <directory>./vendor</directory>
```
Comment (stof) [inline phpunit.xml.dist]: this is useless now, as the vendor folder is outside the source folder

Comment (odolbeau) [reply phpunit.xml.dist]: fixed

**Pull request merged**